scapy/layers/dot11.py

```diff
@@ -339,8 +343,12 @@ def post_build(self, p, pay):
                     icv = ""
                 else:
                     icv = p[4:8]
-                c = ARC4.new(self.iv+key)
-                p = p[:4]+c.encrypt(pay)+icv
+                e = Cipher(
```

Comment:
I wonder if we should _protect_ the "WEP tests" UT in `regression.uts` in order to make the unit tests run smoothly is cryptography is not installed. What do you think ?